### ISSUE TITLE
The build resources can not be found.

### ISSUE THREAD

**Issue by vlikin** [n3-charts/line-chart#400]
The n3-line-chart has been updated up to v2 by bower. The folder build with resources does not exist. I compiled it manually.

**Comment by chaosmail** [comment]
Hi @vlikin 

tl;dr: `npm install n3-charts` will do the job

In the new version v2 we decided to move to npm for distribution. This gives us more flexibility to decide which parts of the app we are distributing. With npm for example, we only distribute the build files, and we can keep our repository clean at the same time - and that's exactly what we are doing.

However, bower still pulls the latest version if you call it - but without the build files. We are currently thinking of mirroring the latest stable build to a 'build' branch, so that you could still use bower to install n3.

Hope this helps! Let me know if you have any further questions 

Best,
Christoph

**Comment by vlikin** [comment]
Thank you. I see your idea. Bower becomes deprecated versus the npm package. I like bower because it provides already built(compiled) solutions. It is good for people who does not familiar with nodejs.Thank you. I see your idea. Bower becomes deprecated versus the npm packager. I like bower because it provides already built solutions.

**Comment by chaosmail** [comment]
technically, using npm should be easier than using bower - as it is installed with node.js automatically. For the user, not much will change - only that you will list your dependencies in the package.json file instead of a bower.json file - which makes also sense in my opinion.
Finally, it comes down to running `npm install n3-charts` instead of `bower install n3-line-chart`.

If you face any problems please don't hesitate contacting me!

**Comment by vlikin** [comment]
Ok. Thank you.

**Comment by jlvdh** [comment]
I'd like to contribute an argument for supporting bower: Right now I'm working on a project that uses bower for all front-end dependencies. I can only switch to V2 if I change the build flow for the entire project which is kind of a pain. I'm imagining there are other people with similar problems.

PS: I really like your work, keep it up!

**Comment by chaosmail** [comment]
Hi @jlvdh 
thanks for your feedback. Do you think it would be help if we provide the build files as an extra branch?
Best,
Christoph

**Comment by jlvdh** [comment]
Hi Christoph, yes this would definitely help for my case at least :+1:

**Comment by coding2012** [comment]
Just throwing our use-case out there. We're currently using bower and npm. Bower is being used to put libraries in www/lib/<package name>. This is configurable in Bower. Seems like NPM does not allow this customization, so we're thinking of putting in a postinstall script to copy the n3-charts to the www/lib folder. This is a cordova project, so the www folder is required in our case.

**Comment by lorem--ipsum** [comment]
Alternatively, you can now use CDNJS to load the JS and CSS files. Dunno if that helps...

**Comment by Supratentorial** [comment]
I have an argument in favour of bower. I use wiredep to automatically inject my dependencies into index.html whenever I install a new package. This relies on bower.json files to declare which files from packages should be included into index.html. I don't believe there is an equivalent for packages installed via npm.

**Comment by jdegger** [comment]
I also automatic injection of bower dependencies, it's fairly annoying having to inject this manually.

**Comment by claudiosousa** [comment]
+1
